test/lib/range_parser_test.rb

```diff
@@ -33,4 +33,26 @@ class RangeParserTest < ActiveSupport::TestCase
     assert_equal (0.1..0.33), RangeParser.parse('0.1   ,   0.33')
   end
 
+  test 'parses garbage format from JSONAPI' do
+    value = ["[11", "13]"]
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.